carsus/util/tests/test_hash.py

```diff
@@ -0,0 +1,15 @@
+import pytest
```

Comment:
I don't like this test so much. I don't know if with future versions of pyarrow and pandas the hash will remain the same.